Makefile

```diff
@@ -13,7 +13,12 @@ export GOROOT=$(BIN_DIR)/go/
 export GOBIN = $(GOROOT)/bin/
 export PATH := $(GOBIN):$(PATH):$(BIN_DIR)
 GOPATH = $(CURDIR)/.gopath
-GOARCH ?= amd64
+UNAME=$(shell uname -m | tr '[:upper:]' '[:lower:]')
```

Comment:
this should go in a separate PR ?

Reply:
oops,
indeed, will drop the change from this pr and create a new one just for this